packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -584,26 +588,29 @@ const start = async zcf => {
     async initOracle(oracleInstance) {
       assert(quoteKit, X`Must initializeQuoteMint before adding an oracle`);
 
+      /** @type {OracleKey} */
+      const oracleKey = oracleInstance || Far('oracleKey', {});
```

Comment:
This seems to be the heart of the matter of this commit. But it's not clear to me why we would lack an `oracleInstance` here.

Reply:
It's an optional argument.

Reply:
ah... and even when it's omitted, you need a unique identity.